packages/components/pie-webc/components/modal.d.ts

```diff
@@ -0,0 +1 @@
+export * from '@justeattakeaway/pie-modal';
```

Comment:
QQ – it may be good to discuss if there's a way of having a centralised way of exporting these from one file (otherwise we're going to have a lot of individual export files). May be something you've tried, but would be good to catch up about.

Reply:
I created DSW-1962 to create an automated way of including all of the components.